src/scarab.js

```diff
@@ -152,6 +171,32 @@ var MapzenScarab = (function () {
   function _onClickGitHub (event) {
     _track('click', 'github', opts.name)
   }
+  // Clicking info button should lead to pop up description to open up
+  // Clicking info button again should lead to description box closing
+  // If no description provided, do not open description box
+
+  function _onClickInfo(event) {
+    var elem = document.getElementById('description')
+    if (elem.style.display === 'none') {
+        elem.style.display = 'block'
+    } else {
+        elem.style.display = 'none'
+    }
+  }
+
+  function _buildDescription() {
+    var elem = document.getElementById('description')
```

Comment:
Great start so far, @mkong0216! I'll have some other suggestions later, but the part we need to take care of here is this `#description` element. This script assumes that it exists on the page, but it might not. We'll have to construct this element and append it to the page, like the rest of scarab.